meta-freescale-3rdparty-mbl/conf/machine/imx7d-pico-mbl.conf

```diff
@@ -45,7 +45,7 @@ KERNEL_IMAGETYPE = "fitImage"
 SPL_BINARY=""
 UBOOT_EXTLINUX = "0"
 UBOOT_CONFIG = ""
-UBOOT_MACHINE = "pico-pi-imx7d_defconfig"
+UBOOT_MACHINE = "pico-imx7d_bl33_defconfig"
```

Comment:
@bryanodonoghue,

Can you please ensure that the following is printed during u-boot initialization?
`SEC0: RNG instantiated`